study.md

```diff
@@ -21,18 +21,53 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Is this correct?: A method is a function that is a property of an object.  Methods can be written as either:
+let obj = {
+  age: function(param1) {
+    code statements;
+  };
+};
+
+or
+
+obj.age(param1) {
+  code statements;
+};
+
+Do both of the above refer to the same method?  Are there any other ways to write a method?
```

Comment:
The first bit of code defines a method called ```age()``` inside of the object ```obj```.  The second snippet just calls ```obj.age()``` with a floating block of code that gets executed afterward.  The two ways of declaring methods are function declarations ```function myFunction() { ... }``` and function expressions ```const myFunction = function myFunction() { ... };```.